keps/sig-auth/3130-kms-observability/README.md

```diff
@@ -81,7 +81,7 @@ A new field will be added to the `EncryptRequest` and `DecryptRequest` structs i
 ```go
 type EncryptRequest struct {
 	// UID is a unique identifier for the request.
-	UID string `protobuf:"bytes,3,opt,name=uid,proto3" json:"uid,omitempty"`
+	UID *string `protobuf:"bytes,3,opt,name=uid,proto3" json:"uid,omitempty"`
```

Comment:
Describe the _why_ behind the pointer and the feature gate.

Reply:
I always wince at this in protobuf.  Is there any real reason we need the pointer in proto for grpc (in json we would use it)?